### PR TITLE
Deck landing page

### PR DESCRIPTION
**Pull request by Guaris**
re: https://github.com/Kong/developer.konghq.com/pull/41
https://deploy-preview-44--kongdeveloper.netlify.app/deck_landing/

image

### CODE REVIEW

File: app/_landing_pages/deck_landing.yaml [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+               Manage Kong Konnect, Kong Gateway, and Kong Ingress Controller configuration declaratively
+  - row: 
+    - column:
+      - type: structured_text
+        config: 
+          blocks: 
+            - type: text
+              text: | 
+                decK is a command line tool that facilitates API Lifecycle Automation (APIOps) by offering a comprehensive toolkit of commands designed to orchestrate and automate the entire process of API delivery. APIOps involves leveraging automation frameworks to streamline and enforce best practices throughout the API lifecycle. This enables developers and operations teams to manage APIs from development through deployment, ensuring consistency, reliability, and speed in API integrations. 
+                decK is one of the many supported tools used to interact with Kong, to learn about other tools review our [tools page](/tools)
```
Comment (lena-larionova) [inline app/_landing_pages/deck_landing.yaml:25]: Should we talk about what decK does more tangibly, in a following structured text section? That is, "decK manages config via state files etc etc.":

```
## How does decK work?

decK operates on state files. decK state files describe the configuration of Kong Gateway. State files store Kong Gateway configuration in a declarative format, letting you manage services, routes, plugins, consumers, and other entities that define how requests are processed and routed through Kong Gateway.

In the background, decK communicates to your Kong Gateways via the Admin API.
```

Or does this info belong elsewhere?

File: app/_landing_pages/deck_landing.yaml [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+      - type: card
+        config:
+          title: Best Practices
+          cta: https://docs.konghq.com/deck/latest/guides/best-practices/
+    - column:
+      - type: card
+        config:
+          title: Changelog
+          cta: https://github.com/kong/deck/blob/main/CHANGELOG.md/
+  - row:
```
Comment (lena-larionova) [inline app/_landing_pages/deck_landing.yaml:129]: I like the idea of splitting the FAQs on a landing page into columns in principle, but the lengths of these answers vary wildly, so we end up with:

Screenshot 2024-08-07 at 2 03 52 PM

Either we should stick to one column width, collapsible FAQs (which is good for consistency anyway), or some of these answers are too long and need to be somewhere else (eg "Kong already has built-in declarative configuration, do I still need decK?") - except I don't know where the somewhere else is.

For now, I vote for one column, and collapsible sections, for consistency with how we're implementing this elsewhere.

File: app/_landing_pages/deck_landing.yaml [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+              text: "**Why the name ‘decK’?**"
+            - type: text
+              text: |
+                It is simple, short, and easy to use in the terminal. It is derived from the combination of words ‘declarative’ and ‘Kong’.
+    - column:
+      - type: structured_text
+        config:
+          blocks:
+            - type: text
+              text: "**Is there a JSON Schema for decK?**"
```
Comment (lena-larionova) [inline app/_landing_pages/deck_landing.yaml:208]: Is this a tutorial 🤔 As in:

Q: Is there a JSON schema for decK? 
A: Yes, The decK team maintains a JSON schema that you can use to validate YAML files on Github. Learn how to use the schema with VScode >

Tutorial at link:

```
The decK team maintains a JSON schema that you can use to validate YAML files on Github. You can use the schema with a text editor to provide JIT YAML validation. For example, to use the JSON schema with VS Code:
               1. Install the Red Hat YAML extension:
                
                      code --install-extension redhat.vscode-yaml
               2. Edit the plugin settings in VS Code:
                       
                      "yaml.schemas": {
                        "https://json.schemastore.org/kong_json_schema.json": [
                            "kong.yml",
                            "kong.yaml"
                        ]
                    }
               3. Verify that it works by opening your existing `kong.yml` or `kong.yaml` file.
 ```

File: app/_landing_pages/deck_landing.yaml (hunk 1 above)
Comment (Guaris) [reply app/_landing_pages/deck_landing.yaml:25]: I thin that belongs here/

**Comment by mheap** [comment]
Merging to get `x_with_y` in to main and provide an example for the new tabs block to be added as a separate PR

**Pull request merged**